x/concentrated-liquidity/swaps_test.go

```diff
@@ -174,6 +178,7 @@ var (
 			expectedTokenIn:            sdk.NewCoin("usdc", sdk.NewInt(10000000000)),
 			expectedTokenOut:           sdk.NewCoin("eth", sdk.NewInt(1820630)),
 			expectedTick:               sdk.NewInt(321055),
+			expectedSqrtPrice:          sdk.MustNewDecFromStr("78.137149196095607129"),
```

Comment:
How is this computed?

Reply:
hrmm this value seems to be a lil off from what i generate in python. Could you please verify? 

https://www.wolframalpha.com/input?i=74.161984870956629487+%2B+%284761322417.810613244228191057067224+%2F+1197767444.955508123222985080%29

Reply:
i'm following these params: 
liquidity: 1197767444.955508123222985080
sqrtPriceCurrent: 74.161984870956629487 (5500) 
amtRemaning: 4761322417.810613244228191057067224

Reply:
just fixed!